src/pages/VenuePage/TemplateWrapper.tsx

```diff
@@ -1,19 +1,18 @@
 import React from "react";
 import { useHistory } from "react-router-dom";
 
+import { Venue, VenueTemplate } from "types/venues";
+
+import { FriendShipPage } from "pages/FriendShipPage";
 import { ArtPiece } from "components/templates/ArtPiece";
-import { AudienceRouter } from "components/templates/Audience/AudienceRouter";
-import { AvatarRouter } from "components/templates/AvatarGrid/Router";
 import { ConversationSpace } from "components/templates/ConversationSpace";
-import FireBarrel from "components/templates/FireBarrel";
 import { JazzbarRouter } from "components/templates/Jazzbar/JazzbarRouter";
 import { PlayaRouter } from "components/templates/Playa/Router";
 import { PartyMapRouter } from "components/templates/PartyMap/PartyMapRouter";
-
-import { FriendShipPage } from "pages/FriendShipPage";
+import { FireBarrel } from "components/templates/FireBarrel";
+import { Audience } from "components/templates/Audience/Audience";
 import { WithNavigationBar } from "components/organisms/WithNavigationBar";
-
-import { Venue, VenueTemplate } from "types/venues";
+import { AvatarGrid } from "components/templates/AvatarGrid";
```

Comment:
Import sorting seems off here?

Why did the `FireBarrel`/`Audience`/`AvatarGrid` `template/`s move to the bottom of the list? `FriendShipPage` is fine at the top since it's a `page/`

Reply:
Maybe I don't remember the pattern correctly. I believe the components and scss were supposed to be at the bottom of the import list, no?

Reply:
I have a plan to fix this in an automated/linted way across the codebase when I get a chance (see https://github.com/sparkletown/internal-sparkle-issues/issues/156) so don't worry too much about this.

The general pattern is 'most distant/big' thing (eg. 3rd party library) to 'most specific thing' (eg. the stylesheet that relates to this component directly). So pages/templates/organisms/molecules would be in that order in between those two 'extremes', etc.